corehq/apps/user_importer/models.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+
+from csv import DictWriter
+from io import StringIO
+
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from corehq.blobs import CODES, get_blob_db
+
+
+class UserUploadRecord(models.Model):
+    domain = models.TextField()
+    status = JSONField(null=True)
+    task_id = models.CharField(max_length=256)
+    date_created = models.DateTimeField(auto_now_add=True)
```

Comment:
User id or username also seems pretty valuable.

Reply:
Definitely, will add that